lib/lunchbot_web/controllers/shared_cart/shared_cart_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule LunchbotWeb.SharedCart.SharedCartController do
+  import Ecto.Query
+
+  def get_lunch_order_for_office(office_id \\ 1, date \\ Date.utc_today()) do
+    filter_lunch_orders_to_today_and_onwards =
+      from(o in Lunchbot.LunchbotData.OfficeLunchOrder, where: o.day >= ^date)
+
+    lunch_orders =
+      Lunchbot.Repo.get_by(Lunchbot.LunchbotData.Office, id: office_id)
+      |> Lunchbot.Repo.preload(
+        office_lunch_orders:
+          {filter_lunch_orders_to_today_and_onwards,
+           [
+             orders: [:menu, order_items: [:item, order_item_options: [:option]]]
```

Comment:
could simplify using the many_to_many relationship between order_items & options -> `orders: [:menu, order_items: [:item, :options]`.  Because the module is named `Options`, you'd need to add the  :join_keys option to the relationship.  It would become, in order_item.ex:
`many_to_many :options, Lunchbot.LunchbotData.Options, [join_through: “order_item_options”, join_keys: [order_item_id: :id, option_id: :id]]`

Reply:
Played around with that for a bit. It would definitely simplify some calls in the end, but I think I'm going to consider it out of scope for now, as a lot of the current codebase and functions access the options as is, and are breaking when this is changed. 

Don't have time to test everything and make sure its good rn, but would make a good ticket for later.

Reply:
👍